src/components/Form/Form.js

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react'
+import Actions from './Form.Actions'
+import Button from '../Button'
+import { classNames } from '../../utilities/classNames'
+
+export class Form extends React.PureComponent {
+  static Actions = Actions
+
+  static defaultProps = {
+    onCancel: () => {},
+    onDestroy: () => {},
+    onSave: evt => {
+      evt && evt.preventDefault()
+    },
+    saveText: 'Save',
+  }
+
+  render() {
+    const {
+      actionDirection,
+      cancelText,
+      children,
+      className,
+      destroyText,
+      onCancel,
+      onDestroy,
+      onSave,
+      saveText,
+    } = this.props
+
+    const componentClassName = classNames('c-Form', className)
+
+    const saveButton = (
+      <Button
+        className="save-button"
+        kind="primary"
+        size="lg"
+        version={2}
+        submit={true}
+      >
+        {saveText}
+      </Button>
+    )
+
+    const cancelButton = cancelText && (
```

Comment:
Should we have saveText, cancelText, and deleteText have default props and instead only render these buttons if the actions (e.g. onSave, onCancel, and onDestroy respectively) are bound?

Reply:
It might be worth keeping a default value for onSave just because the utility seems to arise from using a form with at least one of the actions.